cms/admin/pageadmin.py

```diff
@@ -705,16 +705,16 @@ def copy_page(self, request, page_id):
         page_languages = page.get_languages()
         site_languages = get_language_list(site_id=site.pk)
 
-        if not any(lang in  page_languages for lang in site_languages):
+        if not any(lang in page_languages for lang in site_languages):
             message = force_str(_("Error! The page you're pasting is not "
-                                   "translated in any of the languages configured by the target site."))
+                                  "translated in any of the languages configured by the target site."))
             return jsonify_request(HttpResponseBadRequest(message))
 
         new_page = form.copy_page()
         return HttpResponse(json.dumps({"id": new_page.pk}), content_type='application/json')
 
     def edit_title_fields(self, request, page_id, language):
-        page = self.get_object(page_id)
+        page = self.get_object(request, page_id)
```

Comment:
```suggestion
        page = self.get_object(request, object_id=page_id)
```

The above suggestion will keep the fix consistent with all other use cases of get_object on this page.

Reply:
@Aiky30 Well spotted!